wolvic/wolvic_content_browser_client.cc

```diff
@@ -25,4 +28,42 @@ WolvicBrowserContext* WolvicContentBrowserClient::browser_context() {
   return browser_main_parts_->browser_context();
 }
 
+std::string WolvicContentBrowserClient::GetUserAgent() {
+  return embedder_support::GetUserAgent();
+}
+
+std::string WolvicContentBrowserClient::GetUserAgentBasedOnPolicy(
```

Comment:
Please add a comment referring to the original implementation in chrome_content_browser_client.cc. This is because, as we rebase, if there are changes upstream, we will want to know where the original code comes from.

I.e. something like this:
```
// Copied from //chrome/browser/chrome_content_browser_client.cc
```

Also, it looks like a good opportunity for refactoring upstream: creating GetUserAgentBasedOnPrefs, that takes prefs service parfameter, and applies this logic, so it can be reused both from chrome and other embedders.

Reply:
Done with 
`// Copied from //content/shell/common/shell_content_client.cc`
because the origin referring code is content/shell/common/shell_content_client.cc.  :)
Thanks!